src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/resources/product_taxon.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Metadata\Operations;
+use Sylius\Resource\Metadata\ResourceMetadata;
+
+return (new ResourceMetadata())
```

Comment:
This one has a small issue on the browser title.
![image](https://github.com/user-attachments/assets/ba254a13-ff81-403c-a4fe-ea79d211e3cb)